### ISSUE TITLE
20: Kernel build errors on gs201 and redbull following e6f816f4

### ISSUE THREAD

**Issue by danielk43** [Divested-Mobile/DivestOS-Build#323]
Have two patches causing problems this time:

gs201
CVE-2022-48862/^5.17
```
../../../../../../kernel/google/gs201/private/gs-google/drivers/vhost/iotlb.c:64:3: error: implicit declaration of function 'vhost_iotlb_add_range_ctx' [-Werror,-Wimplicit-function-decla
ration]
                vhost_iotlb_add_range_ctx(iotlb, start, mid, addr, perm, opaque);
                ^
../../../../../../kernel/google/gs201/private/gs-google/drivers/vhost/iotlb.c:64:3: note: did you mean 'vhost_iotlb_add_range'?
../../../../../../kernel/google/gs201/private/gs-google/drivers/vhost/iotlb.c:49:5: note: 'vhost_iotlb_add_range' declared here
int vhost_iotlb_add_range(struct vhost_iotlb *iotlb,
    ^
../../../../../../kernel/google/gs201/private/gs-google/drivers/vhost/iotlb.c:64:60: error: use of undeclared identifier 'opaque'
                vhost_iotlb_add_range_ctx(iotlb, start, mid, addr, perm, opaque);
                                                                         ^
2 errors generated.
```

CVE-2024-40911/^6.10
```
../../../../../../kernel/google/gs201/private/gs-google/net/wireless/util.c:2019:2: error: implicit declaration of function 'wiphy_lock' [-Werror,-Wimplicit-function-declaration]
        wiphy_lock(&rdev->wiphy);
        ^
../../../../../../kernel/google/gs201/private/gs-google/net/wireless/util.c:2019:2: note: did you mean 'pipe_lock'?
../../../../../../kernel/google/gs201/private/gs-google/include/linux/pipe_fs_i.h:237:6: note: 'pipe_lock' declared here
void pipe_lock(struct pipe_inode_info *);
     ^
../../../../../../kernel/google/gs201/private/gs-google/net/wireless/util.c:2021:2: error: implicit declaration of function 'wiphy_unlock' [-Werror,-Wimplicit-function-declaration]
        wiphy_unlock(&rdev->wiphy);
        ^
../../../../../../kernel/google/gs201/private/gs-google/net/wireless/util.c:2021:2: note: did you mean 'wiphy_lock'?
../../../../../../kernel/google/gs201/private/gs-google/net/wireless/util.c:2019:2: note: 'wiphy_lock' declared here
        wiphy_lock(&rdev->wiphy);
        ^
2 errors generated.
```

redbull
CVE-2024-40911/^6.10
```
../../../../../../kernel/google/redbull/net/wireless/util.c:1925:2: error: implicit declaration of function 'wiphy_lock' [-Werror,-Wimplicit-function-declaration]
        wiphy_lock(&rdev->wiphy);
        ^
../../../../../../kernel/google/redbull/net/wireless/util.c:1925:2: note: did you mean 'pipe_lock'?
../../../../../../kernel/google/redbull/include/linux/pipe_fs_i.h:168:6: note: 'pipe_lock' declared here
void pipe_lock(struct pipe_inode_info *);
     ^
../../../../../../kernel/google/redbull/net/wireless/util.c:1927:2: error: implicit declaration of function 'wiphy_unlock' [-Werror,-Wimplicit-function-declaration]
        wiphy_unlock(&rdev->wiphy);
        ^
../../../../../../kernel/google/redbull/net/wireless/util.c:1927:2: note: did you mean 'wiphy_lock'?
../../../../../../kernel/google/redbull/net/wireless/util.c:1925:2: note: 'wiphy_lock' declared here
        wiphy_lock(&rdev->wiphy);
        ^
2 errors generated.
```

**Comment by SkewedZeppelin** [comment]
these were all fixed in 1f650534953b1efd0d22b35625f4d984bd1097c9

**Comment by danielk43** [comment]
Ah, forgot to rebase. Thanks for the fixes
